site/it/tutorials/customization/basics.ipynb

```diff
@@ -1,480 +1,480 @@
+        "tensor = tf.multiply(ndarray, 42)\n",
+        "print(tensor)\n",
+        "\n",
+        "\n",
+        "print(\"And NumPy operations convert Tensors to numpy arrays automatically\")\n",
+        "print(np.add(tensor, 1))\n",
+        "\n",
+        "print(\"The .numpy() method explicitly converts a Tensor to a numpy array\")\n",
+        "print(tensor.numpy())"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "PBNP8yTRfu_X"
+      },
+      "source": [
+        "## Accelerazione GPU\n",
+        "\n",
+        "Molte operazione di TensorFlow sono accelerate usando la GPU per la computazione. In assenza di annotazioni, TensorFlow decide automaticamente se usare la GPU o la CPU per un'operazione—copiando il tensore tra la memoria CPU e quella della GPU se necessario. I Tensori prodotti da un'operazione sono tipicamente supportate dalla memoria del device su cui l'operazione ha eseguito, per esempio:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "cellView": "code",
+        "colab": {},
+        "colab_type": "code",
+        "id": "3Twf_Rw-gQFM"
+      },
+      "outputs": [],
+      "source": [
+        "x = tf.random.uniform([3, 3])\n",
+        "\n",
+        "print(\"Is there a GPU available: \"),\n",
+        "print(tf.config.experimental.list_physical_devices(\"GPU\"))\n",
+        "\n",
+        "print(\"Is the Tensor on GPU #0:  \"),\n",
+        "print(x.device.endswith('GPU:0'))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "vpgYzgVXW2Ud"
+      },
+      "source": [
+        "### Nomi dei Device\n",
+        "\n",
+        "La proprietà `Tensor.device` fornisce una string interamente qualificata con il nome del device che ospita il contenuto del tensore. Questo nome codifica molti dettagli, tra cui un identificativo dell'indirizzo di rete del host su cui questo programma sta eseguendo e il device dentro quel host. Questo è richiesto per l'esecuzione distribuita di un programma TensorFlow. La stringa termina con `GPU:<N>` se il tensore è piazzato nel `N`-esimo GPU del host."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "ZWZQCimzuqyP"
+      },
+      "source": [
+        "\n",
+        "\n",
+        "### Placement esplicito su un Device\n",
+        "\n",
+        "In TensorFlow, il termine *placement* si refersce a come operazioni individuali sono assegnate a un device per l'esecuzione. Come menzionato, quando non c'è una guida esplicita fornita, TensorFlow decide automaticamente su quale device eseguire un'operazione e copia il tensore su quel device se necessario. Tuttavia le operazione di TensorFlow possono essere esplicitamente assegnati a device specifici usando il context manager `tf.device`, per esempio:"
```

Comment:
device --> dispositivo
quando non c'è una guida esplicita fornita --> quando non è fornita un'istruzione esplicita
possono essere esplicitamente assegnati a device specifici --> possono essere esplicitamente assegnate a specifici dispositivi

Reply:
All these changes are ok for me